src/layouts/sidebar-layout/sidebar-layout.module.css

```diff
@@ -31,5 +33,8 @@
 }
 
 .mainArea {
-	flex: 1 1 0;
+	flex-grow: 1;
+	flex-shrink: 1;
+	min-width: 0;
+	overflow: hidden;
```

Comment:
Missed some "too-wide-content-in-main-area" scenarios in the initial `SidebarLayout` work, correcting it here! 👍